### DISCUSSION TITLE
Total grants spent in 2022 - Anyone can help please?

### DISCUSSION THREAD

**Discussion by Sixto5** [snapshot-labs/snapshot#3449] [Q&A]
Hello everyone and thank you for creating this space to share questions and directions.

My quest is to understand better the grants industry so would like to please know where and how I can find the total grants spent in 2022.  I did my best to find a source to uncover this data, without luck!

I would really appreciate if anyone could help or share a source? 

Thank you,
Renzo

**Comment by ChaituVR** [comment]
@Sixto5 This is Snapshot FAQ, not related to grants

**Reply by Sixto5** [reply]
Thank you so much for your input @ChaituVR - Where would you suggest to share this question?
